yarn-project/end-to-end/src/fixtures/cross_chain_test_harness.ts

```diff
@@ -356,11 +322,4 @@ export class CrossChainTestHarness {
     const unshieldReceipt = await unshieldTx.wait();
     expect(unshieldReceipt.status).toBe(TxStatus.MINED);
   }
-
-  async stop() {
```

Comment:
Should you not still keep the stop, just without the aztecNode now?

Reply:
no `teardown()` from e2e setup is enough!